package.json

```diff
@@ -80,11 +81,11 @@
     "rollup-plugin-replace": "^2.0.0",
     "rollup-plugin-strip-banner": "^0.2.0",
     "semver": "^5.5.0",
+    "shelljs": "^0.8.3",
```

Comment:
#carmackno We need fewer deps. This is unnecessarily complicated.

Reply:
Sure, I'll just delete that last commit then

Reply:
For context I would have just used `exec` or whatever but something something Windows contributors whatever I don't care enough it's CI anyway

Reply:
This was a bit of a last minute Friday drive-by review. The dependency is just one issue. The other issue is its reliance on file system information in general. We should leave that to jest since it's something that does a lot of complex work around invalidation, file watching, module resolution etc. It also fairly frequently changes.

So without this diff, it just runs more than it needs to on CI wasting cycles but should be equivalent otherwise?

Reply:
Yeah